src/lib/pages/podcast-gallery/ui/index.svelte

```diff
@@ -26,3 +42,24 @@
     <PodcastList podcasts={$podcasts} />
   {/if}
 </div>
+
+<!--
+TODO: this is only to show the snackbar in action.
+  Remove after connecting the snackbars with other components
+-->
+<div class="flex px-4 py-2 gap-2">
```

Comment:
Should this really be merged into `main`? I don't think it would look nice in the presentation in 2 days